keyboards/keychron/q2_pro/info.json

```diff
@@ -0,0 +1,62 @@
+{
+    "keyboard_name": "Keychron Q2 Pro",
+    "manufacturer": "Keychron",
+    "url": "https://github.com/Keychron",
+    "maintainer": "lokher",
+    "encoder": {
+        "rotary": [
+            {"pin_a": "A10", "pin_b": "A0", "resolution": 4}
+        ]
+    },
+    "processor": "STM32L432",
+    "bootloader": "stm32-dfu",
+    "usb": {
+    	"vid": "0x3434"
+    },
+    "features": {
+        "bootmagic": true,
+        "mousekey": true,
+        "extrakey": true,
+        "nkro": true,
+        "dip_switch": true,
+        "raw": true,
+        "encoder": true,
```

Comment:
```suggestion
        "encoder": true,
        "encoder_map": true,
```

Reply:
Is there a reason why this should be enabled at the keyboard level instead of the keymap level?

Reply:
I saw it in q1_pro and q3_pro